web/groups.go

```diff
@@ -119,31 +117,19 @@ func (s *QuickFeedService) updateGroup(ctx context.Context, sc scm.SCM, request
 		return fmt.Errorf("failed to get %s repository for group %q: %w", course.Code, group.Name, err)
 	}
 	if repo == nil {
-		// no group repository exists; create new repository for group
-		if request.Name != "" && newGroup.ScmTeamID < 1 {
-			// update group name only if team not already created on SCM
-			newGroup.Name = request.Name
-		}
-		repo, team, err := createRepoAndTeam(ctx, sc, course, newGroup)
+		repo, err := createRepo(ctx, sc, course, newGroup)
 		if err != nil {
 			return err
 		}
 		s.logger.Debugf("Creating group repo in the database: %+v", repo)
```

Comment:
Perhaps we should instead log two events:
`Created group repo on SCM: %+v`
and
`Created group repo in database: %+v`

(meaning we log the successful events after the creation instead of saying `Creating`.)